lib/api/apiUtils/object/versioning.js

```diff
@@ -401,6 +403,18 @@ function overwritingVersioning(objMD, metadataStoreParams) {
     metadataStoreParams.creationTime = objMD['creation-time'];
     metadataStoreParams.lastModifiedDate = objMD['last-modified'];
     metadataStoreParams.updateMicroVersionId = true;
+
+    // update restore
+    const days = objMD.archive?.restoreRequestedDays;
+    const now = Date.now();
+    metadataStoreParams.archive = {
+        archiveInfo: objMD.archive?.archiveInfo,
+        restoreRequestedAt: objMD.archive?.restoreRequestedAt,
+        restoreRequestedDays: objMD.archive?.restoreRequestedDays,
+        restoreCompletedAt: new Date(now),
+        restoreWillExpireAt: new Date(now + (days * oneDay)),
+    };
```

Comment:
I don't think it is needed here.
It would be redundant with: https://github.com/scality/cloudserver/pull/4539/files#diff-a76cecfcb0b408028fb3c7e02359a47730668d6424f8b02bb9522c162c21ce47R173

Reply:
Right, thanks